eva-persistence-jdbc/src/main/kotlin/com/razz/eva/persistence/jdbc/executor/JdbcQueryExecutor.kt

```diff
@@ -84,4 +89,17 @@ class JdbcQueryExecutor(
     override fun getConstraintName(ex: DataAccessException): String? {
         return ex.getCause(PSQLException::class.java)?.serverErrorMessage?.constraint
     }
+
+    private fun DSLContext.using(connection: Connection): DSLContext {
+        val newContext = DSL.using(connection, settings())
+        newContext.configuration().set(QueryTracingListenerProvider(openTelemetry))
+        return newContext
+    }
+
+    private fun span(name: String) = Span.current()?.let {
+        openTelemetry
+            .getTracer("eva")
```

Comment:
How it will work if there is already configured tracer?

Reply:
it is like a fabric - it will get or create a new one